docs/test-drive.md

```diff
@@ -1,6 +1,6 @@
 # Take a test drive
 
-You can easily try out Netlify CMS by running it on a pre-configured starter site. Our example in the intro is the [Kaldi small business Hugo template](https://github.com/netlify-templates/kaldi-hugo-cms-template). Use the deploy button below to build and deploy your own copy of the repository:
+You can easily try out Netlify CMS by running it on a pre-configured starter site. Our example in the [intro](https://github.com/netlify/netlify-cms/blob/master/docs/intro.md) is the [Kaldi small business Hugo template](https://github.com/netlify-templates/kaldi-hugo-cms-template). Use the deploy button below to build and deploy your own copy of the repository:
```

Comment:
@erquhart If I recall correctly, there were a couple of places in one or two of the other doc files (don't recall which off hand but I'll look) that had the URL as I changed it. Should all of them be the relative urls at this point? If so, I'll go through and change them before I make a new pull request with the your (and others') requested changes.